src/test/java/org/testeditor/web/xtext/index/FileBasedXtextIndexFillerTest.xtend

```diff
@@ -0,0 +1,41 @@
+package org.testeditor.web.xtext.index
+
+import java.io.File
+import javax.inject.Inject
+import org.junit.Test
+import org.testeditor.tcl.dsl.TclStandaloneSetup
+
+import static org.assertj.core.api.Assertions.assertThat
+
+class FileBasedXtextIndexFillerTest extends AbstractTestInjectingMembers {
+
+	@Inject
+	var FileBasedXtextIndexFiller indexFiller // class under test
+
+	@Test
+	def void testRegisteredLanguagesAreRelevant() {
+		// given
+		new TclStandaloneSetup().createInjectorAndDoEMFRegistration
+
+		// when
+		val allAccepted = #['one.tcl', 'two.tcl', 'one.config', 'one.tml'] //
```

Comment:
there are two hanging slashes?

Reply:
I added those to prevent the pretty print formatter (that comes with xtend) to split the line at an arbitrary position, completely obfuscating the statements